src/api/kube.ts

```diff
@@ -1404,6 +1404,52 @@ export class KubeHelper {
     }
   }
 
+  async getOAuthClientAuthorizations(clientName: string): Promise<string[]> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+    try {
+      const { body } = await customObjectsApi.listClusterCustomObject('oauth.openshift.io', 'v1', 'oauthclientauthorizations')
+
+      if (!body.items) {
+        return []
+      }
+      const oauthClientAuthorizations = body.items as any[]
+      return oauthClientAuthorizations.filter(o => o.metadata.name && o.clientName === clientName)
```

Comment:
pls don't check `o => o.metadata.name` if we don't need it here.
do this check when it is needed (removing the object)